BDArmory/UI/BDAEditorArmorWindow.cs

```diff
@@ -1039,7 +1039,7 @@ private void CalculateArmorStats()
                 */
                 //armorValue = ProjectileUtils.CalculatePenetration(30, newCaliber, 0.388f, 1109, ArmorDuctility, ArmorDensity, ArmorStrength, 30, 0.8f, false);
                 armorValue = ProjectileUtils.CalculatePenetration(30, 1109, 0.388f, 0.8f, ArmorStrength, ArmorVfactor, ArmorMu1, ArmorMu2, ArmorMu3); //why is this hardcoded? it needs to be the selected armor mat's vars
-                relValue = Mathf.Round(steelValue / armorValue * 10) / 10;
+                BDAMath.RoundToUnit(steelValue / armorValue, 0.1f)
```

Comment:
I'm fairly sure you didn't mean to also remove the `relValue = ` and `;` parts of this line.